Build/PackageFiles/ExtensionBundle/ExtensionBundlePreviewTemplates-4.x.nuspec

```diff
@@ -141,6 +141,9 @@
     <file src="..\..\..\Functions.Templates\templates\TimerTrigger-JavaScript\**" target="templates\TimerTrigger-JavaScript" />
     <file src="..\..\..\Functions.Templates\templates\TimerTrigger-PowerShell\**" target="templates\TimerTrigger-PowerShell" />
     <file src="..\..\..\Functions.Templates\templates\TimerTrigger-Python\**" target="templates\TimerTrigger-Python" />
-    <file src="..\..\..\Functions.Templates\templates\TimerTrigger-TypeScript\**" target="templates\TimerTrigger-TypeScript" />   
+    <file src="..\..\..\Functions.Templates\templates\TimerTrigger-TypeScript\**" target="templates\TimerTrigger-TypeScript" /> 
+
+	  <file src="..\..\..\Functions.Templates\templates\SqlInputBinding-JavaScript\**" target="templates\SqlInputBinding-JavaScript" />
```

Comment:
fix spacing

Reply:
I have it formatted on my local copy and if I try to fix this by formatting, it's formatting the entire document, so I left it as is.

Reply:
Just undo your changes and modify the line manually, you don't need to run format on the whole file. 

(you can even do this from Github if you want)

Reply:
I also use a tool called [SourceTree](https://www.sourcetreeapp.com/) that lets you stage changes line-by-line so you can pick out what changes you want to push up.